content/v2/openapi.yml

```diff
@@ -3973,6 +4110,13 @@ components:
           - title: Non-UK Individual (representing self)
             value: FIND
             description: Non-UK Individual (representing self)
+    ExtendedAttributes:
```

Comment:
What do you think about changing the name of this schema to `ContactExtendedAttributes`? Since we already have extended attributes for the TLDs.